testing/testcafe/tests/scheduler/dragAndDrop/T1118059.ts

```diff
@@ -0,0 +1,110 @@
+import { Selector } from 'testcafe';
+import url from '../../../helpers/getPageUrl';
+import Scheduler from '../../../model/scheduler';
+import createWidget from '../../../helpers/createWidget';
+
+fixture`T1118059`
+  .page(url(__dirname, './pages/T1118059.html'));
+
+test('After drag to draggable component, should be called onAppointmentDeleting event only', async (t) => {
+  const scheduler = new Scheduler('#container');
+
+  await t
+    .dragToElement(scheduler.getAppointment('Regular test app').element, Selector('#drag-container'));
+
+  await t
+    .expect(Selector('#event-text-box').innerText)
+    .eql('onAppointmentDeleting');
+}).before(async () => createWidget('dxScheduler', {
+  dataSource: [{
+    text: 'All day test app 1',
+    startDate: new Date(2021, 3, 26),
```

Comment:
Maybe, we should use the UTC format of the new Date in our tests.
If we do this, we've less trouble with timezones in the future.
Like this:
```js
startDate: new Date('2021-04-26T00:00:00Z'),
// or
startDate: '2021-04-26T00:00:00Z',
```